shared/AppInsightsCommon/src/Util.ts

```diff
@@ -363,7 +363,7 @@ export const CorrelationIdHelper: ICorrelationIdHelper = {
         if (includedDomains) {
             let matchExists: boolean;
             arrForEach(includedDomains, (domain) => {
-                const regex = new RegExp(domain.toLowerCase().replace(/\./g, "\.").replace(/\*/g, ".*"));
+                const regex = new RegExp(domain.toLowerCase().replace(/\\/g, "\\\\").replace(/\./g, "\\.").replace(/\*/g, ".*"));
```

Comment:
Was not escaping passed in \ and was replacing "." with "." rather than changing to "\."